app/main/posts/views/filters/active-filters.directive.js

```diff
@@ -48,6 +48,8 @@ function ActiveFilters($translate, $filter, PostFilters, _, TagEndpoint, RoleEnd
             var activeFilters = angular.copy(filters);
             rawFilters = angular.copy(filters);
 
+            // Remove set filter as it is only relevant to collections and should be immutable in that view
+            delete activeFilters.set;
```

Comment:
this is nicer than expected.. I forgot the collections filter doesn't exist in the dropdown any more :)